test/fixtures_stage_based_messaging.js

```diff
@@ -0,0 +1,867 @@
+                    "active": 'true'
+                }
+            },
+            "response": {
+                "code": 200,
+                "data": {
+                    "count": 0,
+                    "next": null,
+                    "previous": null,
+                    "results": []
+                }
+            }
+        },
+
+        // 1: get active subscriptions for cb245673-aa41-4302-ac47-00000001002
+        {
+            "request": {
+                "url": 'http://sbm.localhost:8001/api/v1/subscriptions/',
+                "method": 'GET',
+                "params": {
+                    "identity": 'cb245673-aa41-4302-ac47-00000001002',
+                    "active": 'true'
+                }
+            },
+            "response": {
+                "code": 200,
+                "data": {
+                    "count": 1,
+                    "next": null,
+                    "previous": null,
+                    "results": [
+                        {
+                            'url': 'http://sbm.localhost:8001/api/v1/subscriptions/51fcca25-2e85-4c44-subscription-1002',
+                            'id': '51fcca25-2e85-4c44-subscription-1002',
+                            'version': 1,
+                            'identity': 'cb245673-aa41-4302-ac47-00000001002',
+                            'messageset': 1,
+                            'next_sequence_number': 1,
+                            'lang': "en",
+                            'active': true,
+                            'completed': false,
+                            'schedule': 1,
+                            'process_status': 0,
+                            'metadata': {},
+                            'created_at': "2016-08-12T06:13:29.693272Z",
+                            'updated_at': "2016-08-12T06:13:29.693272Z"
+                        }
+                    ]
+                }
+            }
+        },
+
+        // 2: get active subscriptions for cb245673-aa41-4302-ac47-00000001003
+        {
+            "request": {
+                "url": 'http://sbm.localhost:8001/api/v1/subscriptions/',
+                "method": 'GET',
+                "params": {
+                    "identity": 'cb245673-aa41-4302-ac47-00000001003',
+                    "active": 'true'
+                }
+            },
+            "response": {
+                "code": 200,
+                "data": {
+                    "count": 1,
+                    "next": null,
+                    "previous": null,
+                    "results": [
+                        {
+                            'url': 'http://sbm.localhost:8001/api/v1/subscriptions/51fcca25-2e85-4c44-subscription-1003',
+                            'id': '51fcca25-2e85-4c44-subscription-1003',
+                            'version': 1,
+                            'identity': 'cb245673-aa41-4302-ac47-00000001003',
+                            'messageset': 200,
```

Comment:
For the django testing I've used certain numberings for the messagesets and schedules. For consistency it would be great if they can corellate - that way it would also be more probable that we'll identify missing sets. Please see here: https://github.com/praekeltfoundation/ndoh-hub/blob/develop/ndoh_hub/utils.py#L219. Please bookmark that file so you can refer to it in the future when more sets get added.